pkg/strategy/common/profit_fixer.go

```diff
@@ -290,23 +423,31 @@ func (f *ProfitFixerBundle) Fix(
 }
 
 type DatabaseProfitFixer struct {
-	tradeService *service.TradeService
-	sessions     map[string]types.ExchangeTradeHistoryService
+	sessions map[string]types.ExchangeTradeHistoryService
 
 	core.ConverterManager
+	*bbgo.Environment
 }
 
-func NewDBProfitFixer(tradeService *service.TradeService) *DatabaseProfitFixer {
+func NewDBProfitFixer(environment *bbgo.Environment) *DatabaseProfitFixer {
```

Comment:
map constructor's name to the struct type name?

Reply:
Ok.
BTW, I make it private and add a universal constructor, `NewProfitFixer()`, which initialize a fixer by the config.